diff --git a/graf2d/asimage/src/libAfterImage/Makefile.in b/graf2d/asimage/src/libAfterImage/Makefile.in
index 99028b13dd..a29b30ad52 100644
--- a/graf2d/asimage/src/libAfterImage/Makefile.in
+++ b/graf2d/asimage/src/libAfterImage/Makefile.in
@@ -78,7 +78,7 @@ CCFLAGS         = @CFLAGS@  @MMX_CFLAGS@
 EXTRA_DEFINES	= @DEFINE_XLOCALE@
 
 RANLIB		= @RANLIB@
-AR		= ar clq
+AR		= ar cq
 CP		= @CP@
 MV		= @MV@
 RM		= @RM@
